### PR TITLE
fix and improve token-program course

### PR DESCRIPTION
**Pull request by wuuer**
### Problem

no try/catch for async/await calls, which is not following CONTRIBUTING.md.
[Token Extensions Program] link is wrong.
[add a second account on devnet] link is wrong.
wrong ts command in the section [Create an Associated Token Account to store the tokens].
console.log displays a wrong explorer link due to "!".

### Summary of Changes

add try/catch for async/await calls following CONTRIBUTING.md.
delete unnecessary "!" for generating a wrong explorer link.
add some missing instructions.
fix a wrong ts command for the section [Create an Associated Token Account to store the tokens].

### CODE REVIEW

File: content/courses/tokens-and-nfts/token-program.md [hunk 1]
```diff
@@ -457,7 +457,7 @@ async function buildTransferTransaction(
 
 ### Lab
 
-We’re going to use the Token Program to create a Token Mint, create an
+We’re going to use the Token Token Program to create a Token Mint, create an
```
Comment (mikemaccana) [inline content/courses/tokens-and-nfts/token-program.md:460]:
```suggestion
We’re going to use the Token Program to create a Token Mint, create an
```

File: content/courses/tokens-and-nfts/token-program.md [hunk 2]
```diff
@@ -617,17 +617,20 @@ const transactionLink = getExplorerLink(
...
   "address",
   tokenMintAccount.toString(),
   "devnet",
 );
 
-console.log(`✅ Look at the token mint again: ${tokenMintLink}!`);
+console.log(`✅ Look at the token mint again: ${tokenMintLink}`);
 ```
 
+Replace `YOUR_TOKEN_MINT_ADDRESS_HERE` with your address of the mint and Run the
```
Comment (mikemaccana) [inline content/courses/tokens-and-nfts/token-program.md:631]:
```suggestion
Replace `YOUR_TOKEN_MINT_ADDRESS_HERE` with your address of the mint and run the
```

File: content/courses/tokens-and-nfts/token-program.md [hunk 3]
```diff
@@ -852,7 +869,7 @@ approval.
 2. Create a new Token Mint in the `CreateMint` component. If you need a
    refresher on how to send transactions to a wallet for approval, have a look
    at the
-   [Wallets lesson](/content/courses/intro-to-solana/interact-with-wallets.md).
+   [Wallets lesson](/developers/courses/intro-to-solana/interact-with-wallets).
```
Comment (mikemaccana) [inline content/courses/tokens-and-nfts/token-program.md:872]: It's weird but links should have .md associated with them - see https://github.com/solana-foundation/developer-content/blob/main/CONTRIBUTING.md#links

Do that for all links.

File: content/courses/tokens-and-nfts/token-program.md [hunk 4]
```diff
@@ -707,14 +710,17 @@ Open the token account in Solana Explorer. Look at the owner - it's the account
...
 token account. Recall that we set the `user` as the `mintAuthority` for the
 `mint` we created.
 
-Create a function `mintTokens` that uses the `spl-token` function `mintTo` to
-mint tokens:
+Create an empty file called `mint-tokens.ts`. Then uses the `spl-token` function
+`mintTo()` to mint tokens. Remember to substitute in your token mint address and
+token account address below!
 
 ```typescript
```
Comment (mikemaccana) [inline content/courses/tokens-and-nfts/token-program.md:725]: Please add filenames to your code examples, see https://github.com/solana-foundation/developer-content/blob/main/CONTRIBUTING.md#file-names

**Review by mikemaccana — COMMENTED** [review]
A few small fixes but we can get this in.

**Comment by mikemaccana** [comment]
Also fix the conflicts - thanks!

**Comment by wuuer** [comment]
> Also fix the conflicts - thanks!

Done

**Comment by mikemaccana** [comment]
Much appreciated thanks @wuuer !

**Pull request merged**